src/tutorial/runtime/introduction.rst

```diff
@@ -1,14 +1,43 @@
-.. _tutorial-cylc-runtime-introduction:
+.. |task-waiting| image:: ../../img/task-job-icons/task-waiting.png
+   :scale: 100%
+   :align: middle
 
-Introduction
-============
+.. |task-submitted| image:: ../../img/task-job-icons/task-submitted.png
+   :scale: 100%
+   :align: middle
```

Comment:
If using these in multiple locations please put them in the `hyperlinks` file so that we don't have to maintain multiple copies.